my-api/src/MyList.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from "react";
+import MyDetail from "./MyDetail.js";
+import { BrowserRouter as Router, Route, Switch, Link } from "react-router-dom";
+
+export default class MyList extends Component {
+  xx = "aaa";
+  state = {
+    datax: [],
+    loading: true,
+    info: []
+    //xx: "aaa"
+  };
+
+  componentDidMount = () => {
+    fetch("https://rickandmortyapi.com/api/character/")
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ datax: data.results });
+        this.setState({ info: data.info });
```

Comment:
You can also combine these into the same `setState` call like so:

`this.setState({ datax: data.results, info: data.info })`